lib/ncbo_annotator.rb

```
@@ -720,18 +733,20 @@ def redis_last_mgrep_restart_default_timestamp()
       def create_term_entry(redis, instance_prefix, ontResourceId, resourceId, label_type, val, semanticTypes)
         begin
           # NCBO-696 - Remove case-sensitive variations on terms in annotator dictionary
-          val.upcase!()
+          val = val.dup.upcase
         rescue ArgumentError => e
+          binding.pry
```

RuboCop (1.73)

[warning] Remove debugger entry point `binding.pry`.

(Lint/Debugger)